### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [ ] check minimal required version of `invenio-base` and `invenio-ext`

**Comment by SamiHiltunen** [comment]
@jirikuncar this seems to be ready for merging

File: setup.py [hunk 1]
```diff
@@ -36,10 +36,12 @@
...
     'Flask>=0.10.1',
     'invenio-accounts>=0.1.2',
-    'invenio-base>=0.2.1',
+    'invenio-base>=0.3.0',
+    'invenio-ext>=0.2.1',
+    'invenio-utils>=0.1.1',
     'mock>=1.0.1',
     'six>=1.7.2',
-    'SQLAlchemy-Utils[encrypted]>=0.30.1',
+    'SQLAlchemy-utils[encrypted]>=0.31.0',
```
Comment (jirikuncar) [inline setup.py]: `SQLAlchemy-utils` vs. `SQLAlchemy-Utils`?

**Comment by jirikuncar** [comment]
- [ ] please remove `   - "travis_retry pip install -e git+https://github.com/inveniosoftware/invenio#egg=invenio"` from `.travis.yml`

**Comment by SamiHiltunen** [comment]
@jirikuncar fixed the points you brought up

File: requirements-devel.txt [hunk 2]
```diff
@@ -17,4 +17,7 @@
 
 -e git+git://github.com/inveniosoftware/invenio-accounts.git#egg=invenio-accounts
 -e git+git://github.com/inveniosoftware/invenio-base.git#egg=invenio-base
+-e git+git://github.com/inveniosoftware/invenio-ext.git#egg=invenio-ext
+-e git+git://github.com/inveniosoftware/invenio-testing.git#egg=invenio-testing
+-e git+git://github.com/inveniosoftware/invenio-utils.git#egg=invenio-utils
```
Comment (jirikuncar) [inline requirements-devel.txt]: `'upgrader' < 'utils'`

**Pull request closed without merging**